### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by delameter**
## Checklist:

- [ ] **I am adding a new extension to a language.**
  - [ ] The new extension is used in hundreds of repositories on GitHub.com
    - Search results for each extension:
      
      - https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.FOOBAR+KEYWORDS
  - [ ] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

- [ ] **I am adding a new language.**
  - [ ] The extension of the new language is used in hundreds of repositories on GitHub.com.
    - Search results for each extension:
      
      -  https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.FOOBAR+KEYWORDS
  - [ ] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a syntax highlighting grammar: [URL to grammar repo]
      
  - [ ] I have added a color
    - Hex value: `#RRGGBB`
    - Rationale: 
  - [ ] I have updated the heuristics to distinguish my language from others using the same extension.

- [ ] **I am fixing a misclassified language**
  - [ ] I have included a new sample for the misclassified language:
    - Sample source(s):
      - [URL to each sample source, if applicable]
    - Sample license(s):
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

- [ ] **I am changing the source of a syntax highlighting grammar**
  - Old: [URL to grammar repo]
  - New: [URL to grammar repo]

- [ ] **I am updating a grammar submodule**
  

- [ ] **I am adding new or changing current functionality**
  
  - [ ] I have added or updated the tests for the new or changed functionality.

- [ ] **I am changing the color associated with a language**
  
  - [ ] I have obtained agreement from the wider language community on this color change.
    - [URL to public discussion]
    - [Optional: URL to official branding guidelines for the language]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,8 @@ lib/linguist.rb
 …
 ```
 
+If you are getting permission errors ("repository path is not owned by current user"), specify your current UID and GID with `-u` option: `-u$(id -u):$(id -g)`.
```
Comment (Alhadis) [inline README.md:215]: This is only peripherally relevant to Linguist, addresses a very specific (and easily diagnosed) scenario, and is arguably something that [Docker's own documentation](https://docs.docker.com/engine/reference/commandline/run/) does a better job of explaining (concerning permission errors raised by `docker run`).

Comment (delameter) [reply README.md:215]: but it's a linguist  who throws this error, not the docker. how can it be "peripherally relevant"?

Comment (Alhadis) [reply README.md:215]: @delameter Linguist doesn't have a `-u` switch, and the command that you're _actually_ passing `-u$(id -u):$(id -g)` to is [`docker run --rm -v`](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/README.md?plain=1#L196) (which in turn runs Linguist, albeit in a controlled environment with permissions that vary depending on what options `docker run` has been executed with.

**Comment by delameter** [comment]
whatever

**Pull request closed without merging**